README.md

```diff
@@ -13,4 +13,51 @@
 
 Objective-C based Network Protocol Testing Framework
 
+# Building this Project
 
+## Minimum requirements for building the project
+
+* Xcode 5 or higher
+* Xcode's Command Line Tools.  From Xcode, install via _Xcode &rarr; Preferences &rarr; Downloads_.
+* xctool: ```brew install -v --HEAD xctool```
+
+## Steps for building this project
+
+0. Clone the repo: ```git clone https://github.com/k3po/k3po.ios.git```
+0. Go to the cloned directory: ```cd k3po.ios```
+0. Build the project: ```xctool -project K3poControl.xcodeproj -scheme Framework```
+
+K3poControl.framework will be available under build/framework folder.
+
+# Using K3poClient in XCTestCase
+
+In a class that extends XCTestCase, a test method can use K3poClient as shown below:
+
+```
+#import <K3poControl/K3poControl.h>
+...
+...
+- (void) testK3poClient {
+    BOOL success = YES;
+
+    // This can be moved into setup method.
+    K3poClient *k3poClient = [[K3poClient alloc] initWithScriptRoot:@"org/kaazing/specification/ws/opening"];
+
+    @try {
+        [k3poClient prepare:@"connection.established/handshake.response"];
+
+         // Code to initialize WebSocket and connect goes here.
+
```

Comment:
It seems like we are trying to test the client against RFC-6455 spec. 
Do you think it might be little confusing to end use to grasp this? May be a short description saying what we are trying to test.